### ISSUE TITLE
 as a voter, i would prefer not having to wait at every election for an email to find and start voting

### ISSUE THREAD

**Issue by KiaraGrouwstra** [benadida/helios-server#367]
presently, there seem to be a few pieces of info relevant to voters to participate in an election:

- user (we can import this so as to stay constant across elections)
- password (potentially held constant using [CAS](https://github.com/benadida/helios-server/blob/master/helios_auth/auth_systems/cas.py))
- election URL (changes per election, there doesn't seem to be a general login where you can find a list of elections you are eligible to participate in)

one challenge we face is the emails occasionally take a bit longer to arrive for some of our users, delaying our voting process during general assemblies.
we would try to account for this by sending the emails in time, then occasionally find the question wording might still change a bit, making this not a great solution.

to address this, we would prefer to ensure we can send voters credentials they can share across elections, as well as a single place to log in where they can find new elections they are eligible to vote in.

is this use-case something that has been considered? is this something handled by CAS?

**Comment by benadida** [comment]
Yes this would be handled by an external authentication provider like CAS.
